server/api/src/rsFileChksum.cpp

```diff
@@ -11,14 +11,32 @@
 #include "irods/irods_hierarchy_parser.hpp"
 #include "irods/MD5Strategy.hpp"
 #include "irods/irods_at_scope_exit.hpp"
+#include "irods/irods_logger.hpp"
 
 #include <fmt/format.h>
 
 #include <algorithm>
 #include <string>
 #include <string_view>
+#include <vector>
+#include <climits>
 
-#define SVR_MD5_BUF_SZ (1024*1024)
+using log_api = irods::experimental::log::api;
+
+static int get_read_buffer_size()
```

Comment:
Consider using an anonymous namespace. But it's fine as-is. Up to you.

Reply:
I put it into an anonymous namespace.